### ISSUE TITLE
Templates with random return value lead to exception 'to many recursions'

### ISSUE THREAD

**Issue by simontaurus** [json-editor/json-editor#1485]
### General information

- json-editor version: 2.10.1

### Expected behavior

Templates generating a random value, e. g. uuid (as workaround for #1481) should work

### Actual behavior

Templates with random return value lead to unhandled exception 'to many recursions' (because the template function is called in an infinite loop).
I guess thats because it is check if the template value is stable by calling the template twice (maybe related #1044), and repeat that if not (what is always true in this case due to the randomness of the template value).

### Steps to reproduce the behavior

Create a schema with a template function 'uuid'

```javascript
{
  "title": "json schema example",
  "type": "object",
  "properties": {
    "example": {
      "type": "string",
      "template": "random_string"
    }
  }
}
```
register a template callback that generates a random string

```javascript
window.JSONEditor.defaults.callbacks.template = {
	"random_string": (jseditor_editor, watched_values) => {
		return crypto.randomUUID();
	},
}
```

A workaround is to use the pseudo-async approach from https://github.com/json-editor/json-editor/issues/1041#issuecomment-903767441

```javascript
window.JSONEditor.defaults.callbacks.template = {
	"random_string": (jseditor_editor, watched_values) => {
		setTimeout(() => {
			jseditor_editor.setValue(crypto.randomUUID());
		}, 100)
		return "";
	},
}
```

**Comment by germanbisurgi** [comment]
fixed in #1489

**Comment by germanbisurgi** [comment]
Fixed in #1489
